packages/flutter/lib/src/widgets/image.dart

```diff
@@ -128,6 +128,8 @@ class Image extends StatefulWidget {
   /// Creates a widget that displays an [ImageStream] obtained from the network.
   ///
   /// The [src], [scale], and [repeat] arguments must not be null.
+  ///
+  /// By default, all network images are cached regardless of HTTP headers.
```

Comment:
Remove "By default"? I don't think users can do anything about it right now.

Reply:
done